BeFake/__main__.py

```diff
@@ -1,7 +1,14 @@
 import json
 import os
+
 from functools import wraps
 
+import string
+import random
+
+from .BeFake import BeFake
```

Comment:
`BeFake` is imported twice now.

Reply:
OMG I didn't see that until now 😂

Reply:
I mean, it probably doesn't break anything. 😄
Note to future readers: Fixed in 9e8ff28e95ae71fd2afd6cb1dd4c4791ed109c9e.